private/buf/bufformat/formatter.go

```diff
@@ -1329,11 +1366,6 @@ func (f *formatter) writeFieldLabel(fieldLabel ast.FieldLabel) {
 	f.WriteString(fieldLabel.Val)
 }
 
-// writeBoolLiteral writes a bool literal (e.g. 'true').
-func (f *formatter) writeBoolLiteral(boolLiteralNode *ast.BoolLiteralNode) {
-	f.WriteString(strconv.FormatBool(boolLiteralNode.Val))
-}
```

Comment:
No such thing as a bool literal in the new version of protocompile. We don't actually know if `true` or `false` is a boolean literal until we do type checking, when interpreting options. It could also be the name of an enum value. So now the AST will just use identifier nodes whose text might be `"true"` or `"false"`.